theano/tensor/tests/test_opt.py

```diff
@@ -3855,15 +3855,15 @@ def test_local_useless_tile(self):
         # But it isn't supported for now, so assert that we raise an
         # error.
```

Comment:
Oh, wait, it is still not supported by the Op.
In the end, I think we should just remove that part, until the end of the test.
In the future, we should add optimizations and tests for the reshape/dimshuffles introduced by the new T.tile, but that's not for now.